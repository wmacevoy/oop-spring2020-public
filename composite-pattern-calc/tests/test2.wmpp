(((((s*9)+(s+815))+p)*((8*(f*6))*((2*4)*(b*b))))*x)
(((8*((t*5)*(h+737)))+(((i+0)+(4+g))*((d*z)+(6*q))))*444)
y
((((9*(k*7))+3)*(((d+5)+(926*f))+((7+t)+(3*d))))*((((6*132)*(g*0))*(c+(0+m)))*((3*0)*a)))
(g*((598+1)+m))
((e*((7+0)+((2+t)*7)))+s)
(((((v+9)*(304*2))*867)+((1*(z*158))*(b*(l*l))))*((((i*281)*(3+y))*j)+(((s*666)*(0+4))+((n*5)+(o+a)))))
(((((5+w)*(7+h))+((1*t)*(0*g)))*((u+(z+v))*((9+b)+(r*b))))+((((m*v)*(2+4))+(j*(l+762)))+(((1*u)*(1+7))*(856+x))))
((w*(9*(y*(v*891))))*((s*((1*4)*1))*((h*(854+p))*(q+(299*6)))))
