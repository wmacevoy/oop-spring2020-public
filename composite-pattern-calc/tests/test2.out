0.0026626023
112933.625
0.008264462
0.0
5.815623
0.21721391
1.10672712E8
73.696884
2.4183638
